README.md

```diff
@@ -28,7 +28,9 @@ diagnostics products can operate together.
 
 ## Reference Implementations
 
-TODO: add link here
+OpenTelemetry provides a reference implementation of the baggage specification. You can find details at [OpenTelemetry API](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/baggage/api.md). OpenTelemetry SDK ships a BaggagePropagator and enables it by default. For example, the .NET version of it is [here](https://github.com/open-telemetry/opentelemetry-dotnet/blob/5ddf9a486e755c53ab73debf87286a934fcbbb51/src/OpenTelemetry.Api/Context/Propagation/BaggagePropagator.cs) and it is documented [here](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Api/README.md#baggage-api).
```

Comment:
To me the word "reference" implies that it implements the entire spec with no changes, additions, or subtractions. I don't think we can say OpenTelemetry is the reference implementation if we don't control it. What if they implement some feature not specified? Is that feature then required?

Reply:
Thanks Daniel for the feedback - updated this to "known implementations". Filed an issue to track writing the test to serve as reference implementation.